src/applications/appeals/995/components/EvidencePrivateRecordsAuthorization.jsx

```diff
@@ -52,19 +71,10 @@ const EvidencePrivateRecordsAuthorization = ({
     },
   };
 
-  /**
-   * We are rendering the va-alert so the focus doesn't need to wait for render
-   * Problems that show up include:
-   * - focusElement will add -1 if this isn't set; and don't make it tabbable
-   *   when hidden
-   * - Only render the alert content since the screenreader can still target
-   *   the headers inside
-   */
-  const isTabbable = hasError ? '0' : '-1';
   return (
     <>
       <form>
-        <va-alert status="warning" visible={hasError} tabIndex={isTabbable}>
+        <va-alert status="warning" visible={hasError}>
           {hasError && authorizationAlertContent(handlers.onAnchorClick)}
         </va-alert>
```

Comment:
Nothing blocking, but I wonder if this has any benefit over something like wrapping the whole alert in a conditional render like this:

```
{ hasError && (
        <va-alert status="warning" tabIndex={isTabbable}>
          authorizationAlertContent(handlers.onAnchorClick)
        </va-alert>
)}
 ```

I have been doing it with the conditional render so that my tests are easier for me to write and can check for the presence of the va-alert altogether... idk

Reply:
Also, you may not need to include a tab index any more. I recently [updated the `focusElement` function to remove the `tabindex="-1"` when the element is blurred](https://github.com/department-of-veterans-affairs/vets-website/pull/23136)